apps/engineeringblog/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,78 @@
+// Addapted from https://amirardalan.com/blog/syntax-highlight-code-in-markdown
+import { FC } from "react";
+import ReactMarkdown, { Components } from "react-markdown";
+import { PrismLight as SyntaxHighlighter } from "react-syntax-highlighter";
+import tsx from "react-syntax-highlighter/dist/cjs/languages/prism/tsx";
+import typescript from "react-syntax-highlighter/dist/cjs/languages/prism/typescript";
+import javascript from "react-syntax-highlighter/dist/cjs/languages/prism/javascript";
+import scss from "react-syntax-highlighter/dist/cjs/languages/prism/scss";
+import bash from "react-syntax-highlighter/dist/cjs/languages/prism/bash";
+import markdown from "react-syntax-highlighter/dist/cjs/languages/prism/markdown";
+import json from "react-syntax-highlighter/dist/cjs/languages/prism/json";
+
+import rangeParser from "parse-numeric-range";
+import { oneDark } from "react-syntax-highlighter/dist/cjs/styles/prism";
```

Comment:
Since we are having both dark and light mode, should we consider making the syntax highlighter switchable maybe between `oneDark` and `oneLight` ?

Reply:
I agree @m453h. Let's do so when we are adding the light/dark mode.

Reply:
I have added #871 to keep track of it.